<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1031</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1031</width>
      <height>0</height>
     </size>
    </property>
    <row>
     <property name="text">
      <string>TÂ° media mensual</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Pp medio mensual</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>ETp0</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Enero</string>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Febrero</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Marzo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Abril</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mayo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Junio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Julio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Agosto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Septiembre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Octubre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Noviembre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Diciembre</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_insertar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insertar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit3">
    <property name="geometry">
     <rect>
      <x>572</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_fila">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Fila</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_columna">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Columna</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_contenido">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Contenido</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_sumar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sumar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_graficar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Graficar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
